### DISCUSSION TITLE
Trying to verify the page with basic authentication

### DISCUSSION THREAD

**Discussion by ZeKam** [stevenvachon/broken-link-checker#250] [Q&A]
Hi, I tried to validate some page with basic auth and I get 'Error: HTML could not be retrieved'
I could see that I am not able to verify some other pages with basic auth also.

I have installed package:
`npm install broken-link-checker -g`
run in cdm:
`blc http://foo:bar@httpbin.org/basic-auth/foo/bar`
Error: HTML could not be retrieved

How can I check the pages using basic auth?

**Comment by ardaOnal** [comment]
Still have the same question, would be great if there is an answer
